### ISSUE TITLE
Understanding the Darcy flow dataset

### ISSUE THREAD

**Issue by adityakalaga** [neuraloperator/neuraloperator#288]
In the **small Darcy flow** example available with the code, the **'x'** tensor in the training dataset **(darcy_train_16.pt)** contains boolean values of True or False. Could you explain what parameter this boolean tensor corresponds to?

**Comment by dhpitt** [comment]
To the best of my understanding, the case is explained in the paper [here](https://arxiv.org/pdf/2111.03794.pdf) on Pg. 11: Darcy's law describes fluid flow through a semi-porous medium (in our case you can think about flow through a cross-section of a rectangular prism). In this case, the input `x` is the function a(x), which is a piecewise function describing a diffusion coefficient over the domain [0,1]x[0,1]. Since it's piecewise and chosen to take on values of 0 or 1 it is represented as a binary mask. Hope that helps!

**Comment by adityakalaga** [comment]
Thanks, @dhpitt, your explanation is really helpful!
